src/pages/Club/Club/ClubFollowers.tsx

```diff
@@ -10,7 +10,9 @@ import Title from 'antd/lib/typography/Title';
 import Spinner from '../../Spinner/Spinner';
 import NotificationBoxApi from '../../../api/NotificationBoxApi';
 import { Modal } from 'antd';
+import addTooltip from '../../../components/Tooltip';
 
+const namesMaxLegth = 21;
```

Comment:
this `21` value appears multiple times
so I guess better option is to declare it right in `addTooltip` component

Reply:
done